vms/platformvm/block/executor/verifier.go

```diff
@@ -86,6 +86,7 @@ func (v *verifier) BanffProposalBlock(b *block.BanffProposalBlock) error {
 	changes.Apply(onAbortState)
 
 	// Apply the changes, if any, from processing the decision txs.
+	// [onCommitState] = [onAbortState] here, either one can be used.
```

Comment:
nit: `[onCommitState] == [onAbortState] here, so either one can be used to construct pending state changes. We apply these changes [TODO: where?] to whichever state is being committed, so the reference to state specified here is not important.`